src/Uno.UWP/Devices/Lights/Lamp.Android.cs

```diff
@@ -113,15 +113,37 @@ private void UpdateLampState()
 			var isOn = _isEnabled && _brightness > 0;
 			lock (_lock)
 			{
-				if ((int)Build.VERSION.SdkInt >= (int)BuildVersionCodes.M)
+				if ((int)Build.VERSION.SdkInt >= (int)BuildVersionCodes.Tiramisu)
```

Comment:
Note: I don't have an Android 13 physical device to test with.

I wrote the code per my understanding of the APIs and per Android documentation.